### ISSUE TITLE
Chrome 79.0.3945.79 - unable to restore tabs

### ISSUE THREAD

**Issue by Smir** [deanoemcke/thegreatsuspender#1070]
Please complete the following information when submitting a feature request or bug report.
 * Extension version: 7.1.0
 * Browser name & version: Chrome Version 79.0.3945.79 (Official Build) (64-bit)
 * Operating system & version: Windows 10 Pro 64-bit

And please also do a search for your request/bug before create a new one thanks!

Yesterday Chrome updated to 79.0.3945.79 and now the addon cannot restore all the tabs. From my 700+ tabs it restores about 15. The tabs are there in the task manager, but not responding as you'd usually except them to.

**Comment by delusionn** [comment]
Smir, I had the same issue, and I'm also a heavy tab user, about half your tabs.  Same issue where processes were running, but not showing visible windows.  I also use two different Chrome users to allow different logins to a few sites I need management and user level experiences on, and I use Session Buddy.  So my first clue that something more than just losing most of my tabs was going on was when SB was telling me I actually DID have ~600 tabs open, since I had used it to try to restore my previous session.

After cleaning up my old session I think what may work for you is this:

- Install Session Buddy if you don't already have a session saver program, since you don't want to lose your tabs.  This isn't permanent.
- Save your current session (you may have to clean up any redundant windows due to this whole mess later).
- Uninstall The Great Suspender extension.
- Close and re-open Chrome.
- Re-install The Great Suspender.
- Use SB to restore your old session if anything went wrong, and uninstall SB once you're done.

When I did this, my prior sessions still saved their suspended status, which was a rather tedious task I wasn't looking forward to repeating.

Not ideal, but better than living without the extension.

**Comment by csis0247** [comment]
Yup, can attest. I lost over 800 tabs because of this. My last saved session was 2 weeks ago, so...

**Comment by delusionn** [comment]
Regarding my suggested fix, it worked for a day, but has recurred, so it's not a permanent solution.

**Comment by q00u** [comment]
There was a known bug in Chrome 79.0.3945.79 where windows were not being rendered. It has been fixed in 79.0.3945.88. Update, restart, see if your session comes back fully.

**Comment by Smir** [comment]
> There was a known bug in Chrome 79.0.3945.79 where windows were not being rendered. It has been fixed in 79.0.3945.88. Update, restart, see if your session comes back fully.

working good, thank you

**Comment by DFNCTSC** [comment]
@Smir @delusionn @csis0247

This is not a problem with The Great Suspender. 

This is a problem with how you manage web content.

600, 700, 800+ tabs is astounding mismanagement, a feat of personal inability to properly use the internet, and **this is not the intended function or purpose of the extension** The Great Suspender.

You need to improve your internet usage habits.

Something that might help you in this journey is OneTab 
 (https://chrome.google.com/webstore/detail/onetab/chphlpgkkbolifaimnlloiipkdnihall?hl=en or https://addons.mozilla.org/en-US/firefox/addon/onetab/ ).

**Comment by delusionn** [comment]
> This is not a problem with The Great Suspender.
> 
> This is a problem with how you manage web content.
> 
> 600, 700, 800+ tabs is astounding mismanagement, a feat of personal inability to properly use the internet, and **this is not the intended function or purpose of the extension** The Great Suspender.
> 
> You need to improve your internet usage habits.
> 
> Something that might help you in this journey is OneTab
> (https://chrome.google.com/webstore/detail/onetab/chphlpgkkbolifaimnlloiipkdnihall?hl=en or https://addons.mozilla.org/en-US/firefox/addon/onetab/ ).
> 
> Or maybe you need to learn how to use bookmarks?

This isn't a comedy forum, your reply isn't helpful or accurate, and this isn't Twitter.  You use your tools I'll use mine.  While 600-800 certainly seems excessive to me for an everyday use, I've certainly been there for temporary projects.  As it turned out, it wasn't a Great Suspender problem, it was a temporary Chrome problem and it _was a problem_ that the Chrome developers acknowledged and resolved.  This is good, because I'd temporarily moved to a different session management software solution and it was sub-par, and did not save session hibernation status between browser restarts.
